### PR TITLE
Adds nodemon and adds dev script

### LINKED ISSUE #14: Make the debug server auto restart on file changes
Use `nodemon` to restart the test server while watching file changes

### PR DESCRIPTION
**Pull request by rajkumaar23**
Fixes #14

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "description": "Covid Stats in Telegram",
   "main": "index.js",
   "scripts": {
-    "start": "node app/index.js",
+    "start": "nodemon app/index.js",
```
Comment (gurrrung) [inline package.json:7]: Thanks, We needed this for development. But I don't want to use `nodemon` in the production environment and `start` command is used in the server to start the app. Can you create a new command like
"dev": "nodemon app/index.js" ?

Comment (rajkumaar23) [reply package.json:7]: Oh yeah, sure.

Comment (rajkumaar23) [reply package.json]: Done ✅

**Comment by gurrrung** [comment]
@rajkumaar23 Thanks, I appreciate the help. Merged.

**Pull request merged**